### ISSUE TITLE
prevent from windows problem.

### ISSUE THREAD

**Issue by byehack** [rootm0s/WinPwnage#70]
https://github.com/rootm0s/WinPwnage/blob/99a4a0d4e299e5530ac28c4a728fc772270698e2/winpwnage/functions/uac/uacMethod4.py#L20

after above cleanup, the windows will go to problem in its environment. so some of the windows features won't work because they don't have `%windir%` value in environment.
you should write its real value in the registry after you got access by this way.

**Comment by rootm0s** [comment]
None of my windows machines has this registry value though, do you know which ones that have it? I guess the easy way is just to empty it once used.

**Comment by byehack** [comment]
> I guess the easy way is just to empty it once used.

better way is get the real value before modify it and then set it again the first value in cleanup functions.

**Comment by byehack** [comment]
> None of my windows machines has this registry value though,

are you sure?
this is the environment value and should be set:
![image](https://user-images.githubusercontent.com/26390254/83378593-64fbf800-a3ee-11ea-9653-aa057a288b2e.png)

**Comment by rootm0s** [comment]
> > None of my windows machines has this registry value though,
> 
> are you sure?
> this is the environment value and should be set:

Yes i am sure.

**Comment by rootm0s** [comment]
I dont think this is a issue, if you edit that value.. Windows will still boot normally. At least my two test VMs are 👍 Can you test and verify

**Comment by rootm0s** [comment]
If this value `WINDIR` is not found under HKEY_CURRENT_USER\Environment the computer will use `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment` instead, where `WINDIR` should be present.

**Comment by byehack** [comment]
> If this value `WINDIR` is not found under HKEY_CURRENT_USER\Environment the computer will use `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment` instead, where `WINDIR` should be present.
> 
> So applications should work normally

yes this is true and i said this method for example.
but it is better to re-set the value to what it was before.

**Comment by rootm0s** [comment]
Send a PR for a fix and i'll review it. Otherwise, ill look into it later in future
